app/gui/src/dashboard/layouts/Drive/Categories/categoriesHooks.tsx

```diff
@@ -92,43 +93,31 @@ export function useCloudCategoryList() {
     queryFn: () => backend.getOrganization(),
   })
 
-  const homeDirectoryId = (() => {
-    switch (user.plan) {
-      case Plan.free:
-      case Plan.solo: {
-        return user.rootDirectoryId
-      }
-      case Plan.team:
-      case Plan.enterprise: {
-        return organization == null ?
-            user.rootDirectoryId
-          : DirectoryId(`directory-${organization.id.replace(/^organization-/, '')}` as const)
-      }
-    }
-  })()
+  const organizationRootId =
+    organization != null ? organizationIdToDirectoryId(organization.id) : user.rootDirectoryId
 
   const cloudCategory: CloudCategory = {
     type: 'cloud',
     id: 'cloud',
     label: getText('cloudCategory'),
     icon: CloudIcon,
-    homeDirectoryId,
+    homeDirectoryId: user.rootDirectoryId,
   }
 
   const recentCategory: RecentCategory = {
     type: 'recent',
     id: 'recent',
     label: getText('recentCategory'),
     icon: RecentIcon,
-    homeDirectoryId,
+    homeDirectoryId: organizationRootId,
```

Comment:
Shouldn't it be null instead, as per our last discussion?

Reply:
you're right, pushed a new commit. highly recommend you check it out just to make sure i'm not accidentally doing something silly